torch/autograd/functional.py

```diff
@@ -408,7 +421,66 @@ def _construct_standard_basis_for(tensors: Tuple[torch.Tensor, ...], tensor_nume
     return chunks
 
 
-def jacobian(func, inputs, create_graph=False, strict=False, vectorize=False):
+def _jacfwd(func, inputs, strict=False, vectorize=False):
+    if strict:
+        raise RuntimeError('torch.autograd.functional.jacobian: `strict=True` '
+                           'and `forward_ad=True` are not supported together (yet). '
+                           'Please either set `strict=False` or '
+                           '`forward_ad=False`.')
+    is_inputs_tuple, inputs = _as_tuple(inputs, "inputs", "jacobian")
+    outputs = func(*inputs)
```

Comment:
Why do you need this? Your jvp function below can also return the output no?

Reply:
There needed to be some more logic to extract the output from the wrapped vmap(dual, but it is fixed now